.github/workflows/commit.yml

```diff
@@ -53,7 +53,9 @@ jobs:
         run: |
           cd python/pyxet
           pip install -r tests/requirements.txt
+          pip install target/wheels/pyxet*.whl
```

Comment:
I remembered this doesn't work in Windows PowerShell (the Windows runner uses PS by default) because wildcard expansion is not supported. Thus the below `Get-ChildItem | Foreach-Object` loop to install.

Reply:
No, I don't...  I'm happy to revert this.  Or just run both maturin build and maturin develop to get the environment correct in both cases.   Let me try it without this line.